db_stress_tool/db_stress_common.h

```diff
@@ -20,6 +20,7 @@
 // NOTE that if FLAGS_test_batches_snapshots is set, the test will have
 // different behavior. See comment of the flag for details.
 
+#include "rocksdb/advanced_options.h"
```

Comment:
Should this be grouped with the other "rocksdb/..." includes in the section below?

Reply:
Fixed